src/components/LandingPageComponent.js

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import image from "../images/imagegrid.png";
+import { Button, Typography } from "@mui/material";
+import { createTheme, ThemeProvider } from "@mui/material/styles";
+import { Grid } from "@material-ui/core";
+import logo from "../images/logo.png";
+const theme = createTheme({
+  palette: {
+    primary: {
+      main: "#FBC02D",
+    },
+  },
+});
+
+function LandingPageComponent() {
+  return (
+    <>
+      {" "}
+      <Grid
+        item
+        style={{ marginTop: 15, marginLeft: 20, position: "absolute" }}
```

Comment:
Inline styles make your component less readable, please move LandingPageComponent.js to LandingPageComponent folder, and add to this folder LandingPageComponent.css file